maxspeed_signals.mss

```diff
@@ -18,6 +18,9 @@
     }
     
     /* German speed signals (Zs 3v) as signs */
+    ["feature"="DE-ESO:zs3v"]["signal_speed_limit_distant_form"="sign"]["signal_speed_limit_distant_speed"=null] {
+      marker-file: url('symbols/de/zs3v-empty-sign-down.svg');
+    }
```

Comment:
Is thre a reason not to add it to the block below?

Reply:
no, should work both ways. Thanks!